services/QuillLMS/engines/evidence/app/models/evidence/rule.rb

```diff
@@ -12,15 +12,16 @@ class Rule < ApplicationRecord
       STATE_ACTIVE = 'active',
       STATE_INACTIVE = 'inactive'
     ]
-    TYPES= [
-      TYPE_AUTOML = 'autoML',
-      TYPE_GRAMMAR = 'grammar',
-      TYPE_OPINION = 'opinion',
-      TYPE_PLAGIARISM = 'plagiarism',
-      TYPE_REGEX_ONE = 'rules-based-1',
-      TYPE_REGEX_TWO = 'rules-based-2',
-      TYPE_REGEX_THREE = 'rules-based-3',
-      TYPE_SPELLING = 'spelling'
+    TYPES = [
+      TYPE_AUTOML       = 'autoML',
+      TYPE_GRAMMAR      = 'grammar',
+      TYPE_OPINION      = 'opinion',
+      TYPE_PLAGIARISM   = 'plagiarism',
+      TYPE_REGEX_ONE    = 'rules-based-1',
+      TYPE_REGEX_TWO    = 'rules-based-2',
+      TYPE_REGEX_THREE  = 'rules-based-3',
+      TYPE_SPELLING     = 'spelling',
+      TYPE_PREFILTER    = 'prefilter'
```

Comment:
Not a huge deal, but it looks like we may be keeping these values in alphabetical order.  I wonder if we should add `TYPE_PREFILTER` just before `TYPE_PLAGIARISM`.